src/lib/loadEvents.ts

```diff
@@ -0,0 +1,33 @@
+import gql from "graphql-tag";
+import { print as printQuery } from "graphql";
+
+import type { ContentfulClient } from "$lib/services/server/contentful";
+import type { EventCollectionQuery } from "./$queries.generated";
+
+type EventItem = NonNullable<EventCollectionQuery["eventEntryCollection"]>["items"][number];
+
+const query = gql`
+  query EventCollection($preview: Boolean = false) {
+    # TODO: A limit of 500 works until it doesn't. We should set up a way to paginate.
```

Comment:
FWIW there are currently `54` events in Contentful.

Reply:
combined with the fact that this is ordered by datetime, I think this should hold up for a while in either case